src/include/sof/math/exp_fcn.h

```diff
@@ -0,0 +1,51 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+#ifndef __SOF_MATH_BASE_EXPONENTIAL_H__
+#define __SOF_MATH_BASE_EXPONENTIAL_H__
+
+#include <rtos/bit.h>
+#include <stdint.h>
+
+#define BIT_MASK_Q62P2 0x4000000000000000
```

Comment:
Can we prefix these macros for SOF maths library. @singalsu do you have suggestions ? e.g.
```SOF_MATH_TRIG_```
@singalsu we need to do this for all maths library code.

Reply:
For this SOF_MATH_EXP_ would be good. Also the macros those are used only by the single C module could be in the module to avoid defining non-useful macros for applications space using exp().

Reply:
__SOF_MATH_EXPONENTIAL_H__  is OK ?